test/code/providers/Container_ImageInventory_Class_Provider_UnitTest.cpp

```diff
@@ -155,14 +155,15 @@ class ImageInventoryTest : public CppUnit::TestFixture
 		// Should have same number of images
 		CPPUNIT_ASSERT_EQUAL(images.size(), context.Size());
 
-		wchar_t currentId[66];
+		wchar_t currentId[72];
 		int imageCount = 0;
 
 		// Verify every field of every object
 		for (unsigned i = 0; i < images.size(); i++)
 		{
 			bool flag = false;
-			mbstowcs(currentId, cJSON_GetObjectItem(images[i], "InstanceID")->valuestring, 65);
+			int rc = mbstowcs(currentId, cJSON_GetObjectItem(images[i], "InstanceID")->valuestring, 71);
+			if (rc > 0 && rc < 72) currentId[rc] = 0;
```

Comment:
Blech, 71 is a magic number. Please change to something like:

```
			int rc = mbstowcs(currentId, cJSON_GetObjectItem(images[i], "InstanceID")->valuestring, sizeof(currentID) - 1);
			if (rc > 0 && rc < sizeof(currentID)) currentId[rc] = 0;
```

This allows for easier changes to the buffer size down the road, should it ever be needed again.